### PR TITLE
Final capstone: frontend

### PR DESCRIPTION
**Pull request by mohamedgamaleldin999999**
Book an Appointment: Complete project
In this pull request we completed the following requirements
The user logs in to the website, by typing the username and password

In the navigation panel, the user can see links to:

Motorcycles/doctors/classes/items that you selected as a theme.
"Reserve" form.
"My reservations".
"Add Doctor (the link is visible to everybody).
"Delete Doctor that you selected as a theme" (the link is visible to everybody).
On the main page, the user can see a list of Doctors.

When the user selects a specific Doctor, they can see the details page with its full description.

In the details page, the user can click the "Reserve" button.
When the user clicks the "Add item" link in the navigation panel they can see a form for adding a new item.

Make the app responsive, creating both mobile and desktop versions.

When the user clicks the "Delete item" link in the navigation panel they can see a list of all items with title and "Delete" button.

When the user clicks the "Delete" button, the selected item is marked as removed and does not show on the main list anymore.
To reserve an appointment, the user has to select a date and city (username and selected item are autofilled).

Use the design based on the ["Book a vespa test-ride"](https://www.behance.net/gallery/26425031/Vespa-Responsive-Redesign/modules/173005583) and add all necessary inputs.
The user can also access the "Reserve" page from the navigation panel. In that case only username is autofilled.
When the user clicks the "My reservations" link in the navigation panel they can see a list of their reservations (with information about item name, date and city).

Add full documentation for your API.

Back-end Pull request
https://github.com/stino-x/doc-appointment-backend/pull/4

### CODE REVIEW

**Review by Meltrust — CHANGES_REQUESTED** [review]
Hi,

Good job so far!
There are some issues that you still need to work on to prepare your project for the final evaluation, but you are almost there!









#### You are really close to finishing the Microverse program!! Keep it up! 👍👍👍

You can do it

### After implementing the requested changes, please submit another review request. ♻️
_Check the comments under the review._







Cheers and Happy coding!👏👏👏

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the previous reviews unless it is requested otherwise._**

------

File: src/Styles/Signup.module.css [hunk 1]
```diff
@@ -0,0 +1,265 @@
...
+
+/* .container {
+  max-width: 800px;
+  background: #fff;
+  height: 85vh;
+  width: 90%;
+  margin-right: auto;
+  margin-left: auto;
+  padding: 25px 70px 10px 70px;
+  box-shadow: 0 0 10px rgba(0, 0, 0, 0.1);
```
[the remark this answers is not in the document]
Comment (stino-x) [reply src/Styles/Signup.module.css]: https://github.com/stino-x/doc-appointment/assets/107426247/e228d76c-09a2-4ee6-bb49-0415ea95c728

@Meltrust

**Review by Meltrust — CHANGES_REQUESTED** [review]
Hi,

Good job so far!
There are some issues that you still need to work on to prepare your project for the final evaluation, but you are almost there!








#### You are really close to finishing the Microverse program!! Keep it up! 👍👍👍

You can do it

### After implementing the requested changes, please submit another review request. ♻️
_Check the comments under the review._







Cheers and Happy coding!👏👏👏

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the previous reviews unless it is requested otherwise._**

------

**Review by Meltrust — APPROVED** [review]
# **Wow, you did it 🎉**
Brilliant
## Thank you for the changes implemented 💪 🥇 ㊗️ 
#### _Unless you want to add more features, go ahead to your final presentation ⏩ ⏩ ⏩_ 
## You are about to finish the Microverse program.  You have come a long way!!!




### Good luck in the software industry!!  I'll see you there. ✨
### **Congratulations!!!!!!**  🎉
applause

### To highlight


- Superb job ✔️  



Cheers and Happy coding!👏👏👏


------

**Pull request merged**